spec/fixtures/files/nomis/get_booking_details_200.json

```diff
@@ -0,0 +1,364 @@
+  },
+  "physicalAttributes": {
+    "gender": "Male",
+    "raceCode": "W1",
+    "ethnicity": "White: Eng./Welsh/Scot./N.Irish/British",
+    "heightFeet": 5,
+    "heightInches": 9,
+    "heightMetres": 1.75,
+    "heightCentimetres": 175,
+    "weightPounds": 183,
+    "weightKilograms": 83,
+    "sexCode": "M"
+  },
+  "physicalCharacteristics": [
+    {
+      "type": "HAIR",
+      "characteristic": "Hair Colour",
+      "detail": "Mouse"
+    },
+    {
+      "type": "R_EYE_C",
+      "characteristic": "Right Eye Colour",
+      "detail": "Hazel"
+    },
+    {
+      "type": "L_EYE_C",
+      "characteristic": "Left Eye Colour",
+      "detail": "Hazel"
+    },
+    {
+      "type": "FACIAL_HAIR",
+      "characteristic": "Facial Hair",
+      "detail": "Clean Shaven"
+    },
+    {
+      "type": "FACE",
+      "characteristic": "Shape of Face",
+      "detail": "Oval"
+    },
+    {
+      "type": "BUILD",
+      "characteristic": "Build",
+      "detail": "Medium"
+    }
+  ],
+  "profileInformation": [
+    {
+      "type": "YOUTH",
+      "question": "Youth Offender?",
+      "resultValue": "No"
+    },
+    {
+      "type": "MARITAL",
+      "question": "Domestic Status",
+      "resultValue": "Single-not married/in civil partnership"
+    },
+    {
+      "type": "SMOKE",
+      "question": "Is the Offender a smoker?",
+      "resultValue": "No"
+    },
+    {
+      "type": "SEXO",
+      "question": "Sexual Orientation",
+      "resultValue": "Heterosexual / Straight"
+    },
+    {
+      "type": "RELF",
+      "question": "Religion",
+      "resultValue": "Jehovah's Witness"
+    },
+    {
+      "type": "NAT",
+      "question": "Nationality?",
+      "resultValue": "British"
+    }
+  ],
+  "physicalMarks": [
+    {
+      "type": "Other",
+      "bodyPart": "Head",
+      "comment": "zjxRmuigysnzjxRmuigys"
+    },
+    {
+      "type": "Other",
+      "bodyPart": "Torso",
+      "comment": "CbCb"
+    }
+  ],
+  "assessments": [
+    {
+      "bookingId": 12345,
+      "classificationCode": "STANDARD",
+      "classification": "Standard",
+      "assessmentCode": "CSR",
+      "assessmentDescription": "CSR Rating",
+      "cellSharingAlertFlag": true,
+      "assessmentDate": "2011-08-31",
+      "nextReviewDate": "2012-08-30",
+      "assessmentStatus": "A",
+      "assessmentSeq": 3,
+      "assessmentComment": "fyRoyPEZXwQShfyRoyPEZXwQS"
+    },
+    {
+      "bookingId": 12345,
+      "classificationCode": "B",
+      "classification": "Cat B",
+      "assessmentCode": "CATEGORY",
+      "assessmentDescription": "Categorisation",
+      "cellSharingAlertFlag": false,
+      "assessmentDate": "2017-02-23",
+      "nextReviewDate": "2018-02-22",
+      "assessmentStatus": "A",
+      "assessmentSeq": 9
+    }
+  ],
+  "csra": "Standard",
+  "category": "Cat B",
+  "categoryCode": "B",
+  "birthPlace": "SCOTLAND",
+  "birthCountryCode": "SCOT",
+  "inOutStatus": "IN",
+  "status": "ACTIVE IN"
+}
```

Comment:
🛑  (no new line eof) same in other fixture

Reply:
fixed 👍